src/main/java/com/okta/tools/helpers/ConfigHelper.java

```diff
@@ -0,0 +1,56 @@
+package com.okta.tools.helpers;
+
+import com.okta.tools.aws.settings.Configuration;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Reader;
+
+public final class ConfigHelper {
+    /**
+     * Gets a reader for the config file. If the file doesn't exist, it creates it
+     *
+     * @return The file reader for the config file
+     * @throws FileNotFoundException
+     * @author Andrei Hava
+     * @since 02/14/2018
+     */
+    public static Reader getConfigReader() throws FileNotFoundException {
+        return FileHelper.getReader(FileHelper.getAwsDirectory().toString() + "/config");
```

Comment:
:bug: It's unsafe to build paths with String concatenation.

Recommendation: move the pre-existing Path-based methods from OktaAwsCliAssumeRole.

Reply:
All of the file path concatenation issues can probably be solved using the strategy I followed here:
https://github.com/oktadeveloper/okta-aws-cli-assume-role/blob/master/src/main/java/com/okta/tools/OktaAwsCliAssumeRole.java#L558-L586

It's a bit more verbose, but I think the explicitness is good and it's hard to work around the conditionally creating folders piece otherwise.